### PR TITLE
Multiple CommonScripts scripts python2 -> python3

### PR DESCRIPTION
**Pull request by Noy-Maimon**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
python2 to python 3

### CODE REVIEW

**Review by yaakovi — APPROVED** [review]

File: Packs/CommonScripts/Scripts/ExtractHTMLTables/ExtractHTMLTables.py [hunk 1]
```diff
@@ -54,6 +54,5 @@ def main():
     demisto.results(extract_html_table(html, indexes))
 
 
-# python2 uses __builtin__ python3 uses builtins
-if __name__ == "__builtin__" or __name__ == "builtins":
+if __name__ == "builtins" or __name__ == '__main__':
```
Comment (tkatzir) [inline Packs/CommonScripts/Scripts/ExtractHTMLTables/ExtractHTMLTables.py:57]:
```suggestion
if __name__ in ['__main__', 'builtin', 'builtins']:
```

File: Packs/CommonScripts/Scripts/ExtractIndicatorsFromTextFile/ExtractIndicatorsFromTextFile.py [hunk 2]
```diff
@@ -2,34 +2,43 @@
...
+
+def main():
+    try:
+        args = demisto.args()
+        demisto.results(extract_indicators_from_file(args))
+    except FileNotFoundError:
+        return_error("File was not found")
+
+
+if __name__ == "builtins" or __name__ == '__main__':
```
Comment (tkatzir) [inline Packs/CommonScripts/Scripts/ExtractIndicatorsFromTextFile/ExtractIndicatorsFromTextFile.py:43]:
```suggestion
if __name__ in ['__main__', 'builtin', 'builtins']:
```

File: Packs/CommonScripts/Scripts/IdentifyAttachedEmail/IdentifyAttachedEmail.py [hunk 3]
```diff
@@ -61,11 +61,15 @@ def main():
...
+    return 'no', None
+
+
+def main():
+    args = demisto.args()
+    result, outputs = identify_attached_mail(args)
+    return_outputs(result, outputs, result)
 
 
-# python2 uses __builtin__ python3 uses builtins
```
Comment (tkatzir) [inline Packs/CommonScripts/Scripts/IdentifyAttachedEmail/IdentifyAttachedEmail.py:69]: if __name__ in ['__main__', 'builtin', 'builtins']:

File: Packs/CommonScripts/Scripts/ReadFile/ReadFile.py [hunk 4]
```diff
@@ -43,12 +24,17 @@ def main():
...
+def main():
     try:
-        demisto.results(main())
+        args = demisto.args()
+        demisto.results(read_file(args))
     except Exception as e:
         return_error('Failed to run script - {}'.format(str(e)))
+
+
+if __name__ == "builtins" or __name__ == '__main__':
```
Comment (tkatzir) [inline Packs/CommonScripts/Scripts/ReadFile/ReadFile.py:39]:
```suggestion
if __name__ in ['__main__', 'builtin', 'builtins']:
```

File: Packs/CommonScripts/Scripts/ReadFile/test_data/test_file.txt [hunk 5]
```diff
@@ -0,0 +1 @@
+abcabcabc
```
Comment (tkatzir) [inline Packs/CommonScripts/Scripts/ReadFile/test_data/test_file.txt:1]: Rename the file to test_file_abc.txt

**Review by tkatzir — APPROVED** [review]
Looks good! Minor comments.

**Pull request merged**